swaap/swaap/Views/PendingContactTableViewCell.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  PendingContactTableViewCell.swift
+//  swaap
+//
+//  Created by Marlon Raskin on 1/2/20.
+//  Copyright © 2020 swaap. All rights reserved.
+//
+
+import UIKit
+
+protocol PendingContactTableViewCellDelegate: AnyObject {
+	func pendingContactRequestAccepted(on cell: PendingContactTableViewCell, contact: ConnectionContact)
+	func pendingContactRequestCancelled(on cell: PendingContactTableViewCell, contact: ConnectionContact)
+}
+
+class PendingContactTableViewCell: UITableViewCell {
+
+	@IBOutlet private weak var boujee: UIImageView!
+	@IBOutlet private weak var nameLabel: UILabel!
+	@IBOutlet private weak var acceptButton: UIButton!
+	@IBOutlet private weak var cancelButton: UIButton!
+
+	var profileController: ProfileController?
+	var connectionContact: ConnectionContact? {
+		didSet {
+			updateViews()
+		}
+	}
+
+	var isIncoming: Bool = true {
+		didSet {
+			updateViews()
+		}
+	}
+
+	weak var delegate: PendingContactTableViewCellDelegate?
+
+    override func awakeFromNib() {
```

Comment:
spaces (looks like this whole file is a mix)

Reply:
Not cool. Honestly don't know how it got like this

Reply:
So GitHub is making this look weird. The file looks perfectly fine. I did fix the bounce name though haha

Reply:
Yeah github doesn’t know how to tab, but spaces will be rendered as a space, so they show up as randomly different. 

You can check for sure by turning on invisibles in Xcode.

Reply:
The harder way is by click drag selection - if indentation highlights space sized chunks instead of tab sized, it’s cuz spaces.